From 844a3a7b77e0ae4fe8d7868ad2b2553d7b04fc4d Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:28:04 +0200
Subject: [PATCH 1/2] fix truncation error

---
 calculator.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/calculator.cpp b/calculator.cpp
index b91afea..34fcf40 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,7 +2,10 @@
 
 int Calculator::Add (double a, double b)
 {
+
     return a + b;
+	return a + b + 0.5;
+
 }
 
 int Calculator::Sub (double a, double b)
-- 
2.33.0

